### DISCUSSION TITLE
Milvus azure aks deployment for v2.3.x

### DISCUSSION THREAD

**Discussion by manas007** [milvus-io/milvus#31426] [Q&A and General discussion]
Following the documentation here : https://milvus.io/docs/azure.md for v2.3.x

Section: Using Azure Blob Storage as external object storage

1. Create storage account and container
2. Get secret key, use the first value
3. Add values.yaml:
        where do you add the values.yaml file? Applying a YAML from the "Create" button in AKS gives the error : "failed to create the 
        'undefined'. Error(400) : Resource kind was not specified"
        
Looks like a section of documentation is missing ?

**Comment by LoveEachDay** [comment]
The `values.yaml` mentioned in the attached documentation is for `helm installation` which will be used in the following section `Deploy Milvus`.
```
helm repo add milvus https://zilliztech.github.io/milvus-helm/
helm repo update
helm install -f values.yaml my-release milvus/milvus
```

**Reply by manas007** [reply]
okay, so we update the values.yaml with the secret key after the helm installation ?

**Reply by manas007** [reply]
![image](https://github.com/milvus-io/milvus/assets/19229097/b7da1663-85a2-4924-9820-593ba25030f6)

**Reply by manas007** [reply]
@LoveEachDay : the documentation is not clear on where to add values.yaml ... please advise.

**Reply by manas007** [reply]
@LoveEachDay Any advise ?

**Reply by LoveEachDay** [reply]
@manas007 you can put `values.yaml` into the current working directory where you run `helm install` command.
